actions/docs-verifier/.editorconfig

```diff
@@ -23,9 +23,6 @@ dotnet_diagnostic.CA2007.severity = none
 # IDE0044: Add readonly modifier
 dotnet_style_readonly_field = true
 
-# Default severity for analyzer diagnostics with category 'Style'
-dotnet_analyzer_diagnostic.category-Style.severity = warning
```

Comment:
I think this is an unnecessary noise.

Reply:
Then remove it, this is your PR after all 😄